packages/cli/package.json

```diff
@@ -34,18 +34,20 @@
     "express": "^4.18.1",
     "find-free-port": "^2.0.0",
     "inquirer": "^8.2.4",
+    "lodash": "^4.17.21",
     "luxon": "^2.4.0",
     "pixelmatch": "^5.3.0",
     "pngjs": "^6.0.0",
     "yargs": "^17.5.1"
   },
   "devDependencies": {
-    "@types/express": "^4.17.14"
+    "@types/express": "^4.17.14",
+    "@types/lodash": "^4.14.186"
   },
   "peerDependencies": {
-    "@alwaysmeticulous/record": "^1.7.6",
-    "@alwaysmeticulous/replay-debugger": "^1.7.6",
-    "@alwaysmeticulous/replayer": "^1.7.6"
+    "@alwaysmeticulous/record": "^2.2.1",
+    "@alwaysmeticulous/replay-debugger": "^2.2.1",
+    "@alwaysmeticulous/replayer": "^2.2.1"
```

Comment:
I bumped the dependencies here since I made breaking changes in the API between these packages and the CLI, and so want to ensure that when people bump the CLI they are forced to bump the other packages too. These versions haven't actually been released yet, I will cut a release after this merges.

Side note: presumably our customers are getting peer dependency errors now if they're on replayer 2.* and cli 2.*, since the dependency was set to `^1.7.6`. I guess these show as warns not errors?